addons/hr_expense/views/hr_expense_views.xml

```diff
@@ -197,6 +197,7 @@
                             <field name="date" attrs="{'readonly': [('sheet_is_editable', '=', False)]}"/>
                             <field name="accounting_date" attrs="{'invisible': ['|', ('accounting_date', '=', False), ('state', 'not in', ['approved', 'done'])]}" />
                             <field name="account_id" options="{'no_create': True}" domain="[('account_type', 'not in', ('asset_receivable','liability_payable','asset_cash','liability_credit_card')), ('company_id', '=', company_id)]" groups="account.group_account_readonly" attrs="{'readonly': ['|', ('is_editable', '=', False), ('sheet_is_editable', '=', False)]}" context="{'default_company_id': company_id}"/>
+                            <field name="account_id" invisible="1" groups="!account.group_account_readonly"/>
```

Comment:
Why is it needed in the view?
If you look at `account.view_move_form`, it is never in the view when you don't have the correct groups, but it still works...

Normally the only reason to need the field in the view is because
* an onchange function depends on it
* an `attrs` (invisible/required/readonly) depends on it